types/src/transaction/mod.rs

```diff
@@ -2011,7 +2407,12 @@ impl AccountOrderedTransactionsWithProof {
                     ledger_version,
                 );
 
-                txn_with_proof.verify_user_txn(ledger_info, txn_version, account, expected_seq_num)
+                txn_with_proof.verify_user_txn(
+                    ledger_info,
+                    txn_version,
+                    account,
+                    ReplayProtector::SequenceNumber(expected_seq_num),
```

Comment:
how does this work with Nonce?

Reply:
`AccountOrderedTransactionsWithProof` only contains ordered transactions (sequence number based transactions). It doesn't contain nonce based transactions.